docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -218,7 +219,7 @@ Don't design your add-in to use multiple task panes if you are planning to use a
 
 ## Give us feedback
 
-We'd love to hear your feedback on this feature. If you find any bugs, issues, or have requests on this feature, please let us know by creating a GitHub issue in the [office-js repo](https://github.com/OfficeDev/office-js).
+We'd love to hear your feedback on this feature. If you find any bugs, issues, or have requests on this feature, please let us know by going to the **Feedback** section at the bottom of the page, then selecting **This page** to create a GitHub issue.
```

Comment:
This should continue to point to the office-js repo because that's where we gather feedback on preview features. the Runtime team is not checking docs for feature feedback.